From 137dafe7f9f173b6a4e112f485b979d0dfb0b767 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Wed, 28 Dec 2022 01:01:37 +0800
Subject: [PATCH 13/28] Revert "drm/drv: Fix potential memory leak in
 drm_dev_init()"

This reverts commit c47a823ea186263ab69cfb665327b7f72cb5e779.
---
 drivers/gpu/drm/drm_drv.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/drm_drv.c b/drivers/gpu/drm/drm_drv.c
index 4ca995ce19af6..006e3b896caea 100644
--- a/drivers/gpu/drm/drm_drv.c
+++ b/drivers/gpu/drm/drm_drv.c
@@ -610,7 +610,7 @@ static int drm_dev_init(struct drm_device *dev,
 	mutex_init(&dev->clientlist_mutex);
 	mutex_init(&dev->master_mutex);
 
-	ret = drmm_add_action_or_reset(dev, drm_dev_init_release, NULL);
+	ret = drmm_add_action(dev, drm_dev_init_release, NULL);
 	if (ret)
 		return ret;
 
-- 
2.25.1

